chart/jenkins-operator/templates/operator.yaml

```diff
@@ -57,6 +57,8 @@ spec:
             - name: WATCH_NAMESPACE
               {{- if .Values.jenkins.enabled }}
               value: {{ .Values.jenkins.namespace }}
+              {{- else if .Values.operator.watchNamespace }}
```

Comment:
Please add this option in values.yaml but commented with a short explanation on how to use it. Thanks